### DISCUSSION TITLE
Using Frozen model to make another model

### DISCUSSION THREAD

**Discussion by SeungJay** [deepmodeling/deepmd-kit#2658] [Q&A]
Hi, I want to do MD simulation for 0.1 M NaCl aqueous solution.

Based on the previous pure water model developed by other people (e.g. H2O model uploaded at AIS square; https://www.aissquare.com/datasets/detail?pageType=datasets&name=H2O-SCAN0), I want to make another model to simulate 0.1 M NaCl solution.

This is the strategy that I thought.

1. Download the frozen model in database.
2. Generate the data which contains H2O and NaCl by myself.
3. To train the model with new data based on the previous model. (i.e. dp train input.json -f previous_model.pb)

However, it seems not working. 
Can you guide me how to use the "-f" option appropriately?

I think it doesn't work even though I make my own model and use it from the pre_trained model with same data using "-f" option. 

Thanks in advance

**Comment by njzjz** [comment]
The model defined in `input.json` needs to be the same as the previous one.
